charts/reporting-operator/values.yaml

```diff
@@ -108,6 +108,14 @@ spec:
         spec:
           promsum:
             query: "node-capacity-cpu-cores"
+      pvc-capacity-bytes:
```

Comment:
The key here is the datasources name, so you'll want to change this to `persistentvolumeclaim-usage-bytes` also, since that's what's used in the `reportDataSources` list of your reportGenerationQuery.